core/state/dump.go

```diff
@@ -37,6 +38,7 @@ type DumpConfig struct {
 	OnlyWithAddresses bool
 	Start             []byte
 	Max               uint64
+	StateScheme       string
```

Comment:
Is it possible that the Dump function is called elsewhere, and is the conf properly configured?

Reply:
`StateScheme` in `DumpConfig` doesn't affect other function.

Reply:
api_debug.go maybe use dump-related func, already checked?

Reply:
api_debug doesn't use this field, do it?

Reply:
Use default hash is ok.

Reply:
But the real trie-db may be path-base.

Reply:
fixed